OpenRA.Mods.Common/ActorExts.cs

```diff
@@ -81,7 +81,7 @@ public static Target ResolveFrozenActorOrder(this Actor self, Order order, Color
 
 			// Target is still alive - resolve the real order
 			if (frozen.Actor != null && frozen.Actor.IsInWorld)
-				return Target.FromActor(frozen.Actor);
+				return Target.FromFrozenActor(frozen);
```

Comment:
Yup, added. Do you know if we have a ticket for that? I couldn't find anything in a quick search.